### PR TITLE
Refactor directory structure

### PR DESCRIPTION
**Pull request by kdada**
**What this PR does**
- fix(*): fix headers for *.sh and *.go
- refactor(*): refactor directory structure
- feat(log): implement a basic std logger

**Special notes for your reviewer**:
The basic logger is very rough. It will be enhanced in the future.

### CODE REVIEW

File: utils/profiling/profiling.go [hunk 1]
```diff
@@ -51,8 +50,8 @@ func pprofHandler(w http.ResponseWriter, r *http.Request) {
 	pprofIndex(w, r)
 }
 
-// Install adds the Profiling webservice to the given mux.
-func (d profiling) Install(s web.Server) {
+// Install adds the Profiling serviceservice to the given mux.
```
Comment (supereagle) [inline utils/profiling/profiling.go]: `serviceservice`, typo?

File: log/std.go [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+func (l *stdLogger) exitIfFatal(severity Severity) {
+	if severity == SeverityFatal {
+		os.Exit(1)
+	}
+}
+
+func (l *stdLogger) write(data string) {
+	if data[len(data)-1] != '\n' {
```
Comment (Jimexist) [inline log/std.go]: what if i logged empty string?

Comment (kdada) [reply log/std.go]: It's impossible.  `data` has `header` at least.
```
INFO  1222-18:01:16.566+08 log_test.go:94 | 
```

Comment (kdada) [reply log/std.go]: It's impossible.  `data` has `header` at least.
```
INFO  1222-18:01:16.566+08 log_test.go:94 | 
```

**Pull request merged**